modules/charts/src/main.tsx

```diff
@@ -2,6 +2,6 @@ import { Barchart } from './components/charts/Barchart/Barchart';
 
 import { Headless } from './components/charts/Headless';
 
-import { ChartsProvider } from './components/Provider';
+import { ChartsProvider } from './components/Provider/Provider';
```

Comment:
not sure how to phrase succinctly without seeming snarky: another "almost" barrel?